<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processSchedulers</class>
 <widget class="QWidget" name="processSchedulers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SJF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FPPS_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>FPPS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="STCF_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>STCF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FIFO_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>FIFO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RR_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>RR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MLFQ_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>MLFQ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LS_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NONE_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurSchedule_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
